### PR TITLE
Added cards on the Integration Collection page

### PR DESCRIPTION
**Pull request by sahilsuman933**
Signed-off-by: Sahil Suman <sahilsuman933@gmail.com>

**Description**
- Added cards on the integration collection page

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e3863ad7bbb80556651ca2fba18490b8befaeb29 at: https://62fe967dbd8b8f11b3ce1af0--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 47a79181595678e2c8b35eb1f008543b8de4eed3 at: https://62fe9eb662b8a5006b583905--layer5.netlify.app

**Review by Nikhil-Ladha — COMMENTED** [review]
Can you please update the `overflow-y: scroll` to `overflow-y: hidden` on the `ConactFormModal` component?

**Comment by Nikhil-Ladha** [comment]
Also, I think the whole card should be clickable, else just having that border transition on-hover could be misleading to the user 🤔 
Thoughts @leecalcote @debo19

**Comment by sahilsuman933** [comment]
@Nikhil-Ladha Done.

**Comment by l5io** [comment]
🚀 Preview for commit f51297092f78e5caf6097cf86fb9f6e9ad1b6089 at: https://62ff27295cd704474153632b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9c0d07e65cbf9d29a93a12c28632f5aaebc1f901 at: https://62ff90e1e40e660ac7360782--layer5.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
Regarding the two graphics used in the cards: we never want to use the icons from that stock flat icon site - I forget the name of it - but these are too generic, too overused, too cheap, etc.

**Comment by leecalcote** [comment]
> @Nikhil-Ladha Done.

The cards are still not clickable. Please use 1px border on hover (not 5px).

**Comment by leecalcote** [comment]
The expanding text animation for the main description of the integration is nice. Regarding "See Documentation -->", it's nice to have the arrow animate on hover. It is undesirable to have "See Documentation -->" animate on page load.


Screen Shot 2022-08-19 at 10 07 58 AM

**Comment by sahilsuman933** [comment]
@leecalcote card is not clickable cause we have yet to decide whether the whole card should be clickable or only the button. I think it would be better to remove the animation on the card and directly apply the border cause it doesn't make sense to have a clickable card here.

Also, when we finalize the SVG icons for these cards, I will update that.

**Comment by leecalcote** [comment]
@sahilsuman933 that's fine: only the button is clickable and the border will be reduced and persistently displayed. That'll work. We're still not linking to the integrations page publicly. Push, push, go, go.

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/individual-integrations.style.js [hunk 1]
```diff
@@ -84,9 +84,9 @@ export const IntegrationPageWrapper = styled.section`
 				align-items: center;
 				a {
 					color: black;
-					transition: all 0.5s ease;
-				}
-				a:hover {
+					transition: padding 0.5s ease;			
+					}
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Meshery-integrations/Individual-Integrations/individual-integrations.style.js:88]:
```suggestion
					transition: padding 0.5s ease;	
                                }
```

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/index.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const IndividualIntegrations = ({ data }) => {
   const { frontmatter, body } = data.mdx;
   return (
     <IntegrationPageWrapper>
-      <section className="herosection">
+      <section className="herosection">``
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Meshery-integrations/Individual-Integrations/index.js:16]:
```suggestion
      <section className="herosection">
```

**Comment by l5io** [comment]
🚀 Preview for commit 5a116f70d0f2fca8d6f3a290172b69faed3378c8 at: https://62ffcd55139e411886e5224b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f3306885718806e6c380302a3d6f16f7eeebecfb at: https://62ffd82d44ba2e2042d4ff00--layer5.netlify.app

File: src/sections/Meshery/Meshery-integrations/Card.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <IntegrationCard>
+      <div className="container">
+        <img src={MissingIntegration} alt="missing integration icon" />
+        <h2>Missing an Integration?</h2>
+        <p>Meshery is an extensible  platform with many purpose-built extension points. Use Meshery’s REST API or its GraphQL API both of which allow you to connect to any Kubernetes-native app.</p>
+        <Button
+          secondary
+          title="Learn about Extension Points"
+          url="#"
```
Comment (leecalcote) [inline src/sections/Meshery/Meshery-integrations/Card.js:18]:
```suggestion
          url="https://docs.meshery.io/extensibility"
```

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/index.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ const IndividualIntegrations = ({ data }) => {
           <h2>Overview</h2>
           <MDXRenderer>{body}</MDXRenderer>
           <section className="external-btns">
-            <Button primary className="get-started" title="Get Started" url="#" />
+            <Button primary className="get-started" title="Get Started" url="https://layer5.io/service-mesh-management/meshery/getting-started" />
```
Comment (leecalcote) [inline src/sections/Meshery/Meshery-integrations/Individual-Integrations/index.js:38]: Ideally, this is a relative path.

Comment (leecalcote) [reply src/sections/Meshery/Meshery-integrations/Individual-Integrations/index.js:38]: with `alt` defined.

**Comment by l5io** [comment]
🚀 Preview for commit 517578b22cee8a6225673bfdf1e63ef0efd5580b at: https://62ffe121613477287fc38861--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**